circle.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           sudo ./install.sh /usr/local
       # other deps
       - run: sudo apt -y update && sudo apt -y install python-pip python2.7 curl unzip
+      # update pip
```

Comment:
Sorry, only intended for you to update the appveryor config since circle was passing. so remove it here 😉

Reply:
@dschep added this as I was trying to make circleci pass again, I'll remove(Also my bad for amending, I can see how that doesn't also help reviews)

For appveyor to pass I did as you suggest it but also updated this https://github.com/UnitedIncome/serverless-python-requirements/pull/312/files#diff-45fdab74782ebb95f53bf5fb376eeffb